<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tytuł zadania</string>
    </property>
   </widget>
   <widget class="QLabel" name="descriptionLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Opis</string>
    </property>
   </widget>
   <widget class="QLabel" name="priorityLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Priorytet</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleInput">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="priorityInput">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateInput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Dodaj zadanie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortPriorityButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Sortuj według priorytetu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortDateButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Sortuj według daty</string>
    </property>
   </widget>
   <widget class="QListWidget" name="taskList">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>230</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="descriptionInput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>304</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24pt;
color: white;</string>
    </property>
    <property name="text">
     <string>PLAN MASTER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExportToTxt">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>471</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapisz do pliku .txt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnImportToTx">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>530</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapisz do pliku .txt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoveTask">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>usun zadanie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCompleteTask">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>zakoncz zadanie</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlanMaster">
    <property name="title">
     <string>PlanMaster</string>
    </property>
   </widget>
   <addaction name="menuPlanMaster"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>priorityInput</tabstop>
  <tabstop>dateInput</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>sortPriorityButton</tabstop>
  <tabstop>sortDateButton</tabstop>
  <tabstop>taskList</tabstop>
  <tabstop>calendarWidget</tabstop>
  <tabstop>titleInput</tabstop>
  <tabstop>btnExportToTxt</tabstop>
  <tabstop>btnImportToTx</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
